### PR TITLE
SERV-172

### PR DESCRIPTION
**Pull request by dshemin**

### CODE REVIEW

File: bytes/json.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ func unquoteBytes(s []byte) (t []byte) { //nolint:gocyclo // It's OK.
 	r := 0
 	for r < len(s) {
 		c := s[r]
-		if c == '\\' || c == '"' || c < ' ' {
```
Comment (add2) [inline bytes/json.go:23]: You need to write in the comments to the function that it has been modified after copying it from the encoding/json package

File: bytes/json.go [hunk 2]
```diff
@@ -146,9 +146,10 @@ func normalizeBytes(s []byte) []byte { //nolint:gocyclo // It's OK.
 			default:
```
Comment (add2) [inline bytes/json.go:157]: How does the unquoteBytes function differ from normalizeBytes. It's not obvious. Do we need both of them? Maybe add a comment about it?

**Review by add2 — CHANGES_REQUESTED** [review]

File: bytes/json.go (hunk 1 above)
Comment (dshemin) [reply bytes/json.go:23]: An appropriate comment was added to both functions.

File: bytes/json.go (hunk 2 above)
Comment (dshemin) [reply bytes/json.go:157]: An appropriate comment was added to both functions.

**Review by add2 — APPROVED** [review]

**Pull request merged**